connectors/analytics/comscore/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+}
+apply from: "$rootDir/connectors/publish.gradle"
+
+android {
+    namespace 'com.theoplayer.android.connector.analytics.comscore'
+    compileSdk 34
+
+    defaultConfig {
+        minSdk 21
+        targetSdk 34
+
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
+dependencies {
+    compileOnly "com.theoplayer.theoplayer-sdk-android:core:$sdkVersion"
+    compileOnly "com.theoplayer.theoplayer-sdk-android:integration-ads-ima:$sdkVersion"
+
+    implementation "com.comscore:android-analytics:6.+"
```

Comment:
Normally we add the third party SDK as a `compileOnly` and then use the `implementation` in the `app/build.gradle`.

Reply:
Done

Reply:
how will this be published to Maven? Who will bring in the dependency? Should we, or do we document it for the customers?

Reply:
There is publish.gradle script that does publishing. It picks up projects and publishes them to maven. @hovig-theo knows more about that, I guess

Reply:
Indeed the `publish.gradle` file prepares the maven publishing which is picked up by the Jitpack.

Regarding the dependencies, the user will have to add it and that gives them the benefit to change the version in their app as long as there's no breaking change from the third-party SDK.